charts/incubator/miniflux/questions.yaml

```diff
@@ -13,10 +13,19 @@ questions:
                                 - variable: env
                                   group: "App Configuration"
                                   label: "Image Environment"
+                                  description: >
+                                    Refer to
+                                    <br /><a href="https://miniflux.app/docs/configuration.html" target="_blank" rel="noopener noreferrer">https://miniflux.app/docs/configuration.html</a>
```

Comment:
env section is generic and usually modified using regex matching etc. Lets not add this descripion.
You can add it on the docs

Reply:
suggestion applied in 3a89517d49908b4c067a466f4114c6e6a1eaa41b.